docs/CHANGELOG-JAVASCRIPT-PLUGIN-API.md

```diff
@@ -7,6 +7,12 @@ in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.16.0] - 2023-10-26
+
+### Added
+
+- Adds support for `component` property when using `addPostAdminMenuButton`. The component must be a component import.
```

Comment:
```suggestion
- Adds support for `component` property when using `addPostAdminMenuButton`. The component must be a component class.
```

(technically no reason for it to be 'imported', it could be defined in the same file using JS or gjs)

Reply:
Also, we should document this new property in `plugin-api.js`